code/go/0chain.net/smartcontract/dbs/event/challenge_pool_test.go

```diff
@@ -0,0 +1 @@
+package event
```

Comment:
should we add tests?

Reply:
Yes, we probably should add some unit tests here.